src/main/java/com/example/bookstoreapp/mapper/BookMapper.java

```diff
@@ -0,0 +1,19 @@
+package com.example.bookstoreapp.mapper;
+
+import com.example.bookstoreapp.config.MapperConfig;
+import com.example.bookstoreapp.dto.BookDto;
+import com.example.bookstoreapp.dto.CreateBookRequestDto;
+import com.example.bookstoreapp.model.Book;
+import org.mapstruct.Mapper;
+
+@Mapper(config = MapperConfig.class)
+public interface BookMapper {
+
+    BookDto bookToBookDto(Book book);
+
+    Book bookDtoToBook(BookDto bookDto);
+
+    CreateBookRequestDto bookToCreateBookRequestDto(Book book);
+
+    Book createBookRequestDtoToBook(CreateBookRequestDto createBookRequestDto);
```

Comment:
I may be wrong, but as I understand it, creating all these methods is unnecessary. It would be better to create the toDto and toEntity methods...